### PR TITLE
Feature/role grafana improvements

### PR DESCRIPTION
**Pull request by paasi6666**
- added optional `grafana__smtp_config`
- added optional `grafana__https_config`
- added `ssl_skip_verify` for ldap config
- fixed attribute username in ldap config
- fixed quotation for grafana admin reset (to use more complex passwords)

### CODE REVIEW

**Review by NavidSassan — CHANGES_REQUESTED** [review]
thanks for your contributions :D

**Comment by NavidSassan** [comment]
please let me know if keeping the `attribute_username` and `attribute_email` variables works for you, then I can adjust and merge

File: roles/grafana/templates/etc/grafana/ldap.toml.j2 [hunk 1]
```diff
@@ -50,9 +50,9 @@ group_search_filter_user_attribute = "{{ grafana__ldap_config.group_search_filte
 [servers.attributes]
 name = "givenName"
 surname = "sn"
-username = "{{ grafana__ldap_config.attribute_username | default('cn') }}"
+username = "{{ grafana__ldap_config.username | default('cn') }}"
 member_of = "memberOf"
-email =  "{{ grafana__ldap_config.attribute_email | default('email') }}"
+email =  "{{ grafana__ldap_config.email | default('email') }}"
```
[the remark this answers is not in the document]
Comment (paasi6666) [reply roles/grafana/templates/etc/grafana/ldap.toml.j2:55]: Makes perfect sense. Thanks!

**Review by paasi6666 — COMMENTED** [review]
Makes perfect sense. Thanks!

**Comment by paasi6666** [comment]
> please let me know if keeping the `attribute_username` and `attribute_email` variables works for you, then I can adjust and merge

Still works after changing the `attribute_username` and `attribute_email` variables.

**Pull request merged**